### ISSUE TITLE
playback groups misbehaving

### ISSUE THREAD

**Issue by jfabernathy** [bennettpeter/android-MythTV-Leanfront#41]
I have Leanfront running on a FireTV 4K.  I have 3 Playback groups; Default, Sports, News.  Default and News are in the same storage group. Sports is in its own.
I've noticed that when I have recordings in Default and News groups and then a Sport recording happens, I get the first one in the Sport group in Leanfront, but all the other Sports recordings only show in the 'ALL' listing.  

Also If I delete what is the only recording in Sports group in leanfront, it is replaced by the next Sports recording.  I see that in Settings I only have 2 Recording Groups.  Is this part of the issue?

**Comment by bennettpeter** [comment]
Did you try the Refresh button in the bottom row? Leanfront cannot detect changes at the backend until there is a refresh. Try refresh while in the Sports group. Refresh will only refresh the group you are currently looking at. On the main screen (the one with MythTV logo at the top), refresh will refresh everything in all groups and videos.

It automatically performs a full refresh once per hour.

**Comment by jfabernathy** [comment]
I did try refreshing but probably at the top level. Not sure how to refresh in a group. This usually only happens on Saturday and Sunday when a lot of sports and news gets recorded

**Comment by bennettpeter** [comment]
Sorry, there is confusion here about recording groups, playback groups and storage groups.

When you first come into leanfront you are in the main page with recording groups down the left. If you refresh there it should refresh everything.

In the main page if you click one of the recording groups listed on the left, you get into a page showing that recording group. If you refresh there, it only refreshes that recording group.

Bottom line, on the page where you expect to see a recording, pressing refresh should show it.

**Comment by bennettpeter** [comment]
I need some more information on how to recreate this condition, and whether the appropriate refresh solves it.

**Comment by jfabernathy** [comment]
I found that figuring this out has been a complicated task.I tried exiting Leanfront and restarting and it still couldn't find some of the missing programs under the sports category for example. I'm trying to figure out how to get into a Group without launching what it thinks is the only program in that Group. Sometimes I can click on Sports and it will take me to a list of all the sports titles and sometimes it just launches what it thinks is the only item in the group.  If I can get into the Sports group and hit refresh it will find the rest of the sports.  I'm wondering if clicking on the sports group is different from using the right arrow to navigate into the sports group.

It's hard to test because it usually happens on Saturday when I may be recording 4 football games at the same time.  Once I get Lean front to find them the problem can't be recreated until the next week.

**Comment by jfabernathy** [comment]
This maybe fixed. It's probably getting used to the fact of where you are when you use refresh.  Today's testing with v0-385.
I started Leanfront after 3 football games had started recording. At the top level on the left I had All, Default, LiveTV, Sports, Videos, and Tools.  If I go down to Sports (which shows a 1 under the name) I see an Icon of the first recording in Sports.  If I right arrow it takes me into Sports, but I only see the one recording Icon of the first recording. 

If instead if I click select on Sports, it takes me into the Sports group and I see all 3 games.

without exiting, I watch one game until it was over and then exited it and still saw only 3 games even though I knew 2 more had started recording.  At that time, while in the Sports group I navigated to tools and clicked refresh.  Now I had a list of all 5 games.

So knowing how it works, I'll not have an issue with this again.  This is one case when right arrow and select don't do the same thing. This is an assumption that using mythfrontend conditions your brain to.
